String/AufgabeE.java

```diff
@@ -0,0 +1,41 @@
+package QC_Java.String;
+
+public class AufgabeE {
+    /*
+        Gegeben sei folgendes Snippet:
+                ...
+                String result = "";
+                for(int i = 0; i < 10000; i++) {
+                    result += generateRandomString()
+                }
+                ...
+        Optimieren Sie diesen Code.
+    */
+    public static String Sprint9_Aufgabe(){
```

Comment:
please rename the method, should start from lower case, should follow camel case and be a verb that reflects what method does.

Reply:
Fixed